src/app/core/auth/auth.selectors.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { selectAuth, selectIsAuthenticated } from './auth.selectors';
+
+const createAuthState = ({ isAuthenticated = false } = {}) => ({
```

Comment:
Move this to the bottom of the page to stay consistent.

Reply:
This itself gives lint error "variable 'createAuthState' used before declaration", unless if I didn't misunderstood you.

Reply:
I think if you encounter this lint error, making it a function would resolve it.